app/assets/javascripts/automation_scenarios.coffee

```diff
@@ -48,6 +48,22 @@ document.addEventListener 'turbolinks:load', ->
       name: name
     }
 
+  buildIntersectionPoints = (intersectionsData) ->
```

Comment:
This looks much better than the JS that used to build these intersection points.